internal/shared/util/image/pull_test.go

```diff
@@ -298,8 +298,15 @@ func buildSourceContextFunc(t *testing.T, ref reference.Named) func(context.Cont
 		require.NoError(t, enc.Encode(registriesConf))
 		require.NoError(t, f.Close())
 
+		// Create an insecure policy for testing to override any system-level policy
+		// that might reject unsigned images
+		policyPath := filepath.Join(configDir, "policy.json")
+		insecurePolicy := `{"default":[{"type":"insecureAcceptAnything"}]}`
+		require.NoError(t, os.WriteFile(policyPath, []byte(insecurePolicy), 0644))
+
 		return &types.SystemContext{
 			SystemRegistriesConfPath: registriesConfPath,
+			SignaturePolicyPath:      policyPath,
```

Comment:
I think we need to here like we here: https://github.com/openshift/operator-framework-operator-controller/blob/main/openshift/default-catalog-consistency/pkg/extract/extract.go#L231-L251

Reply:
Thanks for the review @camilamacedo86!
What's the concern here?  Is it that we're dropping a file in t.TempDir() for use in this test scenario? 
That's the only difference I see between the approaches.

Reply:
I'm not following you.  
Right now this test fails on users' machines with a restrictive signing policy, but the test should not be sensitive to users' environments. 
[Here](https://github.com/operator-framework/operator-controller/pull/2339/files#diff-10967a567bc639d23ffbfaed1334bfe5627ffd23ad3d980b94a289a0cfd493cdR304) creates an insecure signing policy for the scoped tests to be used instead of detecting the users' policies.